subsys/bluetooth/host/keys.c

```diff
@@ -371,3 +412,26 @@ SETTINGS_STATIC_HANDLER_DEFINE(bt_keys, "bt/keys", NULL, keys_set, keys_commit,
 			       NULL);
 
 #endif /* CONFIG_BT_SETTINGS */
+
+#if defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST)
+void bt_keys_notify_usage(u8_t id, const bt_addr_le_t *addr)
+{
+	if (!addr) {
+		return;
+	}
+	struct bt_keys *keys = bt_keys_find_addr(id, addr);
+	if (!keys) {
+		return;
+	}
+	if (last_keys_updated == keys) {
+		return;
+	}
+	keys->aging_counter = ++aging_counter_val;
+	last_keys_updated = keys;
+	BT_DBG("Aging counter for %s is set to %u", bt_addr_le_str(addr),
+	       keys->aging_counter);
```

Comment:
Add an empty line here also to not have so much unrelated code in a single big monolithic chunk (helps readability)